back/app/controllers/omniauth_callback_controller.rb

```diff
@@ -124,30 +116,66 @@ def auth_callback(verify:, authver_method:)
       begin
         @user.save!
         SideFxUserService.new.after_create(@user, nil)
-        set_auth_cookie(provider: provider)
-        handle_sso_verification(auth, @user) if verify
-        signup_success_redirect
+        verify_and_sign_in(auth, @user, verify, sign_up: true, user_created: true)
       rescue ActiveRecord::RecordInvalid => e
         Rails.logger.info "Social signup failed: #{e.message}"
-        failure
+        signin_failure_redirect
```

Comment:
I was also wondering why this was renamed? Below, we have two similarly named methods:
- `signin_success_redirect`
- `signup_success_redirect`

Those two do exactly the same thing, except overwrite the `omniauth_params['sso_flow']` parameter with a different value. But for this method, there is no mention of the `sso_flow` parameter- given how the other methods are named, I would expect this one to also overwrite it (and maybe I would also expect there to be a `signup_failure_redirect`). 
Not a big issue of course but was curious about why it was renamed in this way

Reply:
I renamed as `failure` is an endpoint method but merely wraps `signin_failure_redirect` so for me it just read better this way. `sso_flow` does not need to be set as I think the value that is passed through never needs to be changed but in the case of the other methods, it shouldn't always be what's been passed, hence why we set it explicitly.